libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -284,15 +284,15 @@ export class ChartJSService {
     // However the stock chart, should have autogenerated labels if no
     // custom labels are supplied.
     const isStockType = type === 'stock';
-    const labels = !dataLabels
+    const labelsToApply = !labels
```

Comment:
Hard to read: Consider using if else and subsequently a single ternary expression.